iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/vector.inl

```diff
@@ -370,15 +378,16 @@ template <typename T, uint64_t Capacity>
 T& vector<T, Capacity>::at_unchecked(const uint64_t index) noexcept
 {
     // AXIVION Next Construct AutosarC++19_03-A5.2.3 : const cast to avoid code duplication
-    // NOLINTNEXTLINE (cppcoreguidelines-pro-type-const-cast)
+    // NOLINTNEXTLINE(cppcoreguidelines-pro-type-const-cast)
     return const_cast<T&>(const_cast<const vector<T, Capacity>*>(this)->at_unchecked(index));
 }
 
 template <typename T, uint64_t Capacity>
 const T& vector<T, Capacity>::at_unchecked(const uint64_t index) const noexcept
 {
     // AXIVION Next Construct AutosarC++19_03-A5.2.4 : Type-safety ensured by template parameter
-    // NOLINTNEXTLINE (cppcoreguidelines-pro-type-reinterpret-cast)
+    // NOLINTJUSTIFICATION User accessible method at() performs bounds check
```

Comment:
I think bound checks have nothing to do with a `reinterpret_cast` usage.

Reply:
@elfenpiff You have to look more carefully above and below ;) Here, two warnings are suppressed:

1) `reinterpret_cast` for Axivion with justification and clang-tidy without duplicate justification
2) Bounds safety issue of working with raw array instead of `std::span`, justification only for clang-tidy